doc/api/crypto.md

```diff
@@ -1687,12 +1687,15 @@ added: v0.11.14
     `crypto.constants`, which may be: `crypto.constants.RSA_NO_PADDING`,
     `RSA_PKCS1_PADDING`, or `crypto.constants.RSA_PKCS1_OAEP_PADDING`.
 - `buffer` {Buffer | TypedArray | DataView}
+- Returns: {Buffer}
```

Comment:
Maybe we can just put the description on the list?

```
- Returns: {Buffer} A new `Buffer` with the decrypted content.
```

Reply:
Right. I was taking reference from https://github.com/nodejs/node/blob/master/doc/api/buffer.md#bufequalsotherbuffer, where we specify a return type in the list and additional description in the later paragraph.

I realised this is actually a little inconsistent, the next function is documented as you have suggested above. I'll add a commit to clean this up, and implement your suggestion.